src/datetimepicker.windows.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ * @format
+ * @flow strict-local
+ */
+'use strict';
+
+import {requireNativeComponent, StyleSheet} from 'react-native';
+import type {WindowsNativeProps, WindowsDatePickerChangeEvent} from './types';
+import * as React from 'react';
+
+const styles = StyleSheet.create({
+  rnDatePicker: {
+    height: 32,
+    width: 150,
+  },
+});
+
+const RNDateTimePickerWindows = requireNativeComponent('RNDateTimePickerWindows');
+
+export default class DateTimePickerWindows extends React.Component<WindowsNativeProps> {
+  static defaultProps: WindowsNativeProps = {
+    dateFormat: 'day month year',
```

Comment:
The current implementation sets the default value to 'dayofweek day month'. I changed it during testing, can remove altogether.

Reply:
Could you validate that the right behavior happens when you run the app in a locale with a different date format?
- different language (to test day/month names)
- different number system (e.g. Arabic numerals)
- different day/month/year order (e.g. Spanish uses ddMMyyyy instead of MMddyyyy)
- different calendar (e.g. Japanese Imperial calendar)

Reply:
@asklar Just changed my region to Poland - can see different time/date formats and translated names **in explorer**, but not in the test app, even after reboot. 
However, the same happens when checking the DatePicker example in RNTester, so I'm not introducing a regression. Can this be a broader issues with our platform?

Reply:
@rectified95 yes, it's possible this is a broader platform issue. Could you please file a bug in our repo with these details/screenshots? we should understand how RN apps expect to handle i18n and what role should RNW play in it. Normally you get these things for free in XAML so I'm surprised we don't see the right things happen :(
@chrisglein FYI

Reply:
opened issue: microsoft/react-native-windows#4791